test.md

```diff
@@ -0,0 +1 @@
+THIS FILE SHOULD BE IGNORED
```

Comment:
- [OPTIONAL] According to the project requirements, you should git ignore the `test.md` file. I can see that you included it in your `.gitignore` file after you pushed it to this repository. To delete it from the repository, kindly follow the instructions in this stack overflow article https://stackoverflow.com/questions/1274057/how-do-i-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore 👍